### PR TITLE
[wheel] use latest manylinux build image

### PR DESCRIPTION
**Pull request by aslonnie**
which contains fixes for centos mirror

### CODE REVIEW

File: ci/docker/manylinux.Dockerfile [hunk 1]
```diff
@@ -11,19 +11,4 @@ ENV BUILDKITE_BAZEL_CACHE_URL=$BUILDKITE_BAZEL_CACHE_URL
...
-#!/bin/bash
-
-# Centos 7 is EOL and is no longer available from the usual mirrors, so switch
-# to https://vault.centos.org
-sed -i 's/enabled=1/enabled=0/g' /etc/yum/pluginconf.d/fastestmirror.conf
-sed -i 's/^mirrorlist/#mirrorlist/g' /etc/yum.repos.d/*.repo
-sed -i 's;^.*baseurl=http://mirror;baseurl=https://vault;g' /etc/yum.repos.d/*.repo
-if [ "${HOSTTYPE}" == "aarch64" ]; then
-  sed -i 's;/centos/7/;/altarch/7/;g' /etc/yum.repos.d/*.repo
-fi
```
Comment (khluu) [inline ci/docker/manylinux.Dockerfile:24]: why are these removed?

Comment (aslonnie) [reply ci/docker/manylinux.Dockerfile:24]: because they are no longer required.

**Review by khluu — APPROVED** [review]
